client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -198,6 +198,12 @@ const wrapSettingsForm = ( getFormSettings ) => ( SettingsForm ) => {
 							path,
 						} );
 						break;
+					case 'jetpack_subscribe_overlay_enabled':
+						trackTracksEvent( 'calypso_settings_subscription_overlay_updated', {
+							value: fields.sm_enabled,
```

Comment:
`sm_enabled` needs changing to `jetpack_subscribe_overlay_enabled`

Reply:
fixed, thanks!